docs/semgrep-supply-chain/overview.md

```diff
@@ -60,6 +60,8 @@ severity levels for GA languages. That means Semgrep Supply Chain can flag all
 your critical/high-severity findings as either reachable or unreachable.
   * If there's a code pattern in the codebase that matches the vulnerability
     definition, the finding is flagged as **reachable**.
+      * A finding is **always reachable** if it's something Semgrep recommends fixing, regardless of what's in the code. In other words, the presence of the dependency is the vulnerability.
```

Comment:
```suggestion
      * A finding is **always reachable** if it's something Semgrep recommends fixing, regardless of what's in the code. In other words, the presence of the dependency is the vulnerability.
```
I checked the app and this is indeed what's in the tooltip. However, I'm pretty sure the original meaning of Always reachable is that there's no other way to fix the vuln except by upgrading (other reachable findings can be fixed by keeping the library but using a different function)

@andy-r2c Would you suggest we keep the upgrade-only bit in the docs? 
![image](https://github.com/semgrep/semgrep-docs/assets/20766840/05845819-f59e-4862-9175-64b9af085ada)

Reply:
I think we should change the language to be more "strongly suggest" than "you must"